source/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -3318,6 +3318,21 @@
 
 /*---------------------------------------------------------------------------*/
 
+/*
+ * ipconfigMULTICAST_MAC_FILTERING
```

Comment:
I commented earlier about the word MULTICAST: in most cases, the hash table can also be used for unicast filtering. The SAME70 has two bits to enable 2 types of filtering:
~~~
#define GMAC_NCFGR_MTIHEN (0x1u << 6) /**< (GMAC_NCFGR) Multicast Hash Enable */
#define GMAC_NCFGR_UNIHEN (0x1u << 7) /**< (GMAC_NCFGR) Unicast Hash Enable */
~~~
I saw the same possiblity on other platforms.

So why not call it `ipconfigMAC_FILTERING`? And how should `pfAddMulticastMAC()` be called?
~~~
pfAddMulticastMAC()    => pfAddAllowedMAC()
pfRemoveMulticastMAC() => pfRemoveAllowedMAC()
~~~

And yes, we can consider to leave out the entire `ipconfigMULTICAST_MAC_FILTERING` item.

Reply:
WOW, during all of our discussions I have been thinking that the hash register can be used either for multicast OR unicast hash matching, but now I see what you mean. I read the datasheet again and actually tried matching a unicast MAC and you are 100% right. I will rename the functions as per your request and will add some more code in them that enables GMAC_NCFGR_MTIHEN and GMAC_NCFGR_UNIHEN when needed.

As for the ipconfigMULTICAST_MAC_FILTERING define. I will rename it and leave it for a few days in case someone comes up with a really compelling reason to keep it. If nobody does, I will gladly get rid of it.